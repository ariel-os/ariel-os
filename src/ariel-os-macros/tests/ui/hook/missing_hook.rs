#![no_main]

// FAIL: missing hook
#[ariel_os::hook]
fn usb_builder() {}
