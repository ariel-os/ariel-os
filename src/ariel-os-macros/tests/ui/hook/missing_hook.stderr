error: custom attribute panicked
 --> tests/ui/hook/missing_hook.rs:4:1
  |
4 | #[ariel_os::hook]
  | ^^^^^^^^^^^^^^^^^
  |
  = help: message: a hook must be specified
